asciidoc/cypher-workflow.adoc

```diff
@@ -383,7 +416,7 @@ Custom types (those not available in the language or standard library) are highl
 ======
 [options="header", cols=","]
 |===
-| Neo4j type        | .NET type
+| Neo4j Cypher Type        | .NET Type
```

Comment:
Is there a reason for capitalising `Type`?

Reply:
Its a name or heading and then you every first letter should be capitalized? Its just a styling issue.